dir.yaml

```diff
@@ -179,15 +179,15 @@
 
 ## Developers & Testers
 
-# - title_en: Plugins & Extensions
-#   title_cn: 插件与扩展
-#   children:
+- title_en: Plugins & Extensions
+  title_cn: 插件与扩展
+  children:
 #     # TODO: add Introduction
 #     # TODO: add plugin development doc
 #     - advanced/plugins
-#     - advanced/hooks
-#     - advanced/webhook
-#     - advanced/lang-exhook
+    - advanced/hooks
+    - advanced/lang
+    - advanced/lang-exhook
```

Comment:
IMO, The `Multi-language support` concept doesn't need to be mentioned.  Just state how to provide the extension ability by exhook. Therefore, the dir seems like to the following
```
- advanced/hooks
- advanced/exhook
```